app/src/main/java/com/mobiledevelopmentfoodapp/dto/Customer.kt

```diff
@@ -4,7 +4,9 @@ package com.mobiledevelopmentfoodapp.dto
 //import androidx.room.PrimaryKey
 import com.google.gson.annotations.SerializedName
 
-data class Customer(var name: String, var phone: String, @SerializedName("id") var customerId:Int = 0 ) {
+data class Customer(var name: String,
+                    var phone: String,
+                    @SerializedName("id") var customerId:Int = 0, ) {
```

Comment:
Changed the formatting of this class header according to https://kotlinlang.org/docs/coding-conventions.html#class-headers.

Reply:
Added in trailing commas to simplify code generation according to https://kotlinlang.org/docs/coding-conventions.html#trailing-commas.